src/api/mock/resetPasswordAPIMock.js

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+import toastr from "toastr";
+import { DELAY } from "../config";
+
+const USERS = [
+  {
+    email: "dariusndubi@gmail.com"
```

Comment:
what is the email for??

Reply:
Let me remove it I set it out to mock user requesting for a reset.